.github/workflows/operator-test.yaml

```diff
@@ -47,6 +47,12 @@ jobs:
           sudo curl -Lo /usr/local/bin/kubectl-kuttl https://github.com/kudobuilder/kuttl/releases/download/v0.11.1/kubectl-kuttl_0.11.1_linux_x86_64
           sudo chmod +x /usr/local/bin/kubectl-kuttl
 
+      - name: Install metrics-server
+        run: |
+          kubectl apply -f https://github.com/kubernetes-sigs/metrics-server/releases/latest/download/components.yaml
+          kubectl patch deployment -n kube-system metrics-server --type "json" -p '[{"op": "add", "path": "/spec/template/spec/containers/0/args/-", "value": --kubelet-insecure-tls}]'
+          kubectl wait --for=condition=available deployment/metrics-server -n kube-system  --timeout=5m
+
```

Comment:
Why is this needed for upgrade?

Reply:
the 0.56.0 Tag of the operator includes new e2e tests that have a dependency on this metrics server.  These commands are copied from https://github.com/open-telemetry/opentelemetry-operator/blob/main/hack/install-metrics-server.sh

Reply:
Ok, sounds good. Thanks for clarifying